self-core-impl/src/main/java/com/selfxdsd/core/tasks/ContractTasks.java

```diff
@@ -63,13 +71,24 @@ public Task register(final Issue issue) {
     @Override
     public Tasks ofProject(final String repoFullName,
                            final String repoProvider) {
-        throw new UnsupportedOperationException("Not implemented yet");
+        final List<Task> ofProject = this.tasks.stream()
+            .filter(t -> t.project().repoFullName().equals(repoFullName)
+                && t.project().provider().equals(repoProvider))
+            .collect(Collectors.toList());
+        return new ProjectTasks(repoFullName, repoProvider, ofProject,
+            this.storage);
```

Comment:
@criske Same as bellow, these are the Tasks of a Contract and the Contract belongs to a single Project.